### ISSUE TITLE
`publish.yml` workflow fails on recent master

### ISSUE THREAD

**Issue by jimblandy** [gfx-rs/wgpu#2620]
Our CI for updating the examples on the `wgpu.rs` web site is [failing](https://github.com/gfx-rs/wgpu/runs/6107574663?check_suite_focus=true) on the current examples.

I updated the `gecko` branch to 0b61a191, and the "Generate JS bindings for WebGPU examples" step failed:

```
  for i in target/wasm32-unknown-unknown/release/examples/*.wasm;
  do
    wasm-bindgen --no-typescript --out-dir target/generated-gpu --web "$i";
  done
  shell: /usr/bin/bash -e {0}
  env:
    RUSTFLAGS: --cfg=web_sys_unstable_apis
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-cli-support-0.2.80/src/descriptor.rs:208:15
```

**Comment by cwfitzgerald** [comment]
This is almost certainly the wasm-bindgen cli mismatching with the version of wasm-bindgen we're using as a library. They must match exactly.

**Comment by jimblandy** [comment]
#2624 should fix this

**Comment by jimblandy** [comment]
Fixed in c226a103.
